### PR TITLE
first draft of publish script

### PR DESCRIPTION
**Pull request by asjohnston-asf**

### CODE REVIEW

**Comment by jtherrmann** [comment]
TODO:
- [ ] Add secrets

File: .github/workflows/publish-products-from-hyp3-to-asf.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  EARTHDATA_PASSWORD: ${{ secrets.EARTHDATA_PASSWORD }}
+  TOPIC_ARN: ${{ secrets.TOPIC_ARN }}
+  RESPONSE_TOPIC_ARN: ${{ secrets.RESPONSE_TOPIC_ARN }}
+
+jobs:
+  publish-products:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
```
Comment (asjohnston-asf) [inline .github/workflows/publish-products-from-hyp3-to-asf.yml:18]: we'll need the `configure-aws-credentials` action here as well, example at https://github.com/ASFHyP3/hyp3/blob/develop/.github/workflows/deploy-daac.yml#L92

Comment (asjohnston-asf) [reply .github/workflows/publish-products-from-hyp3-to-asf.yml:18]: We'll only need `sns:publish` permssions for the ingest-test-jobs and ingest-prod-jobs topics in the GRFN account, so I'm open to ideas for which credentials to use. Also an opportunity to try out using the GitHub OIDC provider and the the `role-to-assume` parameter?

File: .github/workflows/publish-products-from-hyp3-to-asf.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          python-version: 3.10
```
Comment (asjohnston-asf) [inline .github/workflows/publish-products-from-hyp3-to-asf.yml:20]: need to run `python -m pip install -r requirements.txt` after creating the environment

**Comment by jtherrmann** [comment]
The main script looks good to go.

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**